assets/js/data/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+	"checkJs": false,
+	"allowJs": false
+  },
```

Comment:
Speaking of which though, is this even needed? The base config already has these set to false right?